CHANGELOG.rst

```diff
@@ -41,6 +41,7 @@ Changed
   trackers are still loaded from pickle but will be dumped as json in any subsequent
   save operations.
 - Event brokers are now also passed to custom tracker stores (using the ``event_broker`` parameter)
+- Updated Rasa X model pull interval to be used and give a warning message if the user has a custom url in the ``endpoints.yml`` file.
```

Comment:
that's two changes. Let's put it in two separate lines. And I'd also bit more verbose, e.g. `if a ``wait_time_between_pulls`` is configured for the model server in the ``endpoints.yml``, this will be used instead of the default one when running Rasa X`

Reply:
ok I reworded this let me know what you think